### PR TITLE
{bio}[foss/2022a] cryoCARE v0.3.0, CSBDeep v0.7.4 /w CUDA 11.7.0 

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)
requires #19167

### CODE REVIEW

File: easybuild/easyconfigs/c/cryoCARE/cryoCARE-0.3.0-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+]
+
+sanity_check_commands = [
+    'cryoCARE_extract_train_data.py --help',
+    'cryoCARE_train.py --help',
+    'cryoCARE_predict.py --help',
+]
+
+moduleclass = 'bio'
+
```
Comment (verdurin) [inline easybuild/easyconfigs/c/cryoCARE/cryoCARE-0.3.0-foss-2022a-CUDA-11.7.0.eb:62]: @ThomasHoffmann77 please remove the end of file blank line to appease the bot.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by ThomasHoffmann77** [comment]
@verdurin How would I fix `Conflict found for dependencies of cryoCARE-0.3.0-foss-2022a-CUDA-11.7.0: TensorFlow-2.11.0-foss-2022a vs TensorFlow-2.11.0-foss-2022a-CUDA-11.7.0`?

**Comment by branfosj** [comment]
> @verdurin How would I fix `Conflict found for dependencies of cryoCARE-0.3.0-foss-2022a-CUDA-11.7.0: TensorFlow-2.11.0-foss-2022a vs TensorFlow-2.11.0-foss-2022a-CUDA-11.7.0`?

CSBDeep has a dependency on the CPU only TF (`TensorFlow-2.11.0-foss-2022a.eb`). So there should be a `CSBDeep-0.7.4-foss-2022a-CUDA-11.7.0.eb` that changes the TF dep to be the CUDA enabled version and then this PR should depend on that.

Also, does this need `tifffile`, as CSBDeep is providing a different version.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
bear-pg0208u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), 1 x NVIDIA NVIDIA A100-SXM4-40GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/c245cc5d9fd2122242d07de54dbe8e48 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**